2018/11.md

```diff
@@ -137,6 +137,23 @@ Supporting materials includes slides, a link to the proposal repository, a link
     | topic | presenter |
     |-------|-----------|
 
+1. Breakout sessions
+
+    If time permits, we can schedule breakout sessions after the regularly scheduled agenda items. l
+    Let's draft breakout topics below, but continue to accept new proposals at the physical meeting.
+    
+    | topic | discussion prompt |
+    |-------|-------|
+    | IDL deep dive | Examine the semantics of WebIDL, observe its mismatches and pain points with JS, and think about what we want for IDL in JS |
+    | JS standard library infrastructure | How should built-in modules work? |
+    | JS standard library contents | What should we put in the JS standard library? |
+    | Functional programming in JS | How should JavaScript evolve to better support functional programming? |
+    | WebAssembly/JS interaction | Review various WebAssembly proposals for interacting with JS (gc, host-binfings, modules) and discuss ways to improve interaction |
```

Comment:
Also:
- Type Reflection for JS API
- BigInt <-> i64

There is a typo in `host-bindings`